interpretation/ww_head.py

```diff
@@ -631,13 +642,10 @@ def process(message, isGameMaster=False, isAdmin=False, isPeasant=False):
             '''follow'''
             # The command that allows the dog to choose a side.
             if is_command(message, ['bark', 'become', 'choose', 'follow']) and user_role == "Dog":
-                target = check.users(message,1,True,True)
-                if not target:
-                    return [Mailbox().respond("**INVALID SYNTAX:**\nPlease make sure to mention a user.\n\n**Tip:** You can also mention their emoji!",True)]
                 disguise = check.roles(message,1)
                 if not disguise:
                     return [Mailbox().respond("**INVALID SYNTAX:** \nPlease make sure to provide a role.")]
-                return [func.disguise(user_id,target[0],disguise[0])]
+                return [func.dog_follow(user_id,disguise[0])]
```

Comment:
why do we do `dog_follow` to disguise? is this intentional?

Reply:
No, we use `dog_follow` to have the dog choose a role. As of now, the dog unintentionally had the same power as the tanner. But now it works.

Reply:
This is a bug fix that is already a thing on the live Werewolf Bot, but it seems the `master` branch hadn't received that update yet. 🤷‍♀️